Header header
geometry_msgs/Vector3 linear_acceleration
geometry_msgs/Vector3 angular_velocity
geometry_msgs/Vector3 magnetometer
geometry_msgs/Quaternion orientation
geometry_msgs/Quaternion orientation_REF
geometry_msgs/Vector3 angular_velocity_measure
geometry_msgs/Vector3 angular_acceleration_measure
geometry_msgs/Vector3 angular_acceleration_model
geometry_msgs/Vector3 linear_acceleration_model
geometry_msgs/Quaternion quaternion_velocity_model